image-builder/README.md

```diff
@@ -14,12 +14,97 @@ We use the Docker's building system and convert it at the end to a disk image th
 [More info](https://github.com/scaleway/image-tools)
 
 
-:warning: **Documentation incoming** :warning:
+## How to build a custom image (using [scw](https://github.com/scaleway/scaleway-cli))
```

Comment:
`How to build a custom image (using scw)` -> `How to build a custom image using scw`